api/src/main/java/io/strimzi/api/kafka/model/ConnectorState.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.api.kafka.model;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonValue;
+
+public enum ConnectorState {
```

Comment:
I would add even just one line of Javadoc here.

Reply:
I added a short javadoc, let me know if it's not what you had in mind